python/pyxir/contrib/target/components/DPUCAHX8L/u280_op_support.py

```diff
@@ -0,0 +1,223 @@
+        on the DPUCAHX8L-u280 target """
+    return is_concat_supported(X, bXs, tXs, channel_parallel=32, bank_depth=4096)
+
+
+@pyxir.register_op_support_check("DPUCAHX8L-u280", "Convolution")
+def conv2d_op_support(X: XLayer, bXs: List[XLayer], tXs: List[XLayer]) -> bool:
+    """ Check whether we can execute the provided Conv2D operator
+        on the DPUCAHX8L-u280 target """
+    return is_conv2d_supported(
+        X,
+        bXs,
+        tXs,
+        channel_parallel=32,
+        bank_depth=4096,
+        max_stride=4,
+        min_stride=1,
+        depthwise_supported=False,
+    )
+
+
+@pyxir.register_op_support_check("DPUCAHX8L-u280", "Conv2DTranspose")
+def conv2d_transpose_op_support(
+    X: XLayer, bXs: List[XLayer], tXs: List[XLayer]
+) -> bool:
+    """Check whether we can execute the provided Conv2DTranspose operator
+       on the DPUCAHX8L-u280 target"""
+    return is_conv2d_transpose_supported(
+        X, bXs, tXs, channel_parallel=32, bank_depth=4096, max_stride=16, min_stride=1
+    )
+
+
+@pyxir.register_op_support_check("DPUCAHX8L-u280", "DPU")
+def DPUCZDX8G_op_support(X: XLayer, bXs: List[XLayer], tXs: List[XLayer]) -> bool:
+    """ Check whether we can execute the provided DPU operator
+        on the DPUCAHX8L-u280 target """
+    return True
+
+
+@pyxir.register_op_support_check("DPUCAHX8L-u280", "Eltwise")
+def eltwise_op_support(X: XLayer, bXs: List[XLayer], tXs: List[XLayer]) -> bool:
+    """Check whether we can execute the provided Eltwise operator
+       on the DPUCAHX8L-u280 target"""
+    return True
+
+
+@pyxir.register_op_support_check("DPUCAHX8L-u280", "Maximum")
+def maximum_op_support(X: XLayer, bXs: List[XLayer], tXs: List[XLayer]) -> bool:
+    """Check whether we can execute the provided Maximum operator
+        on the DPUCAHX8L-u280 target
+    Return true if part of leaky relu pattern    
+    """
+    # check whether part of leaky relu
+    return "patterns" in X.attrs and "LeakyReLU" in X.attrs["patterns"]
+
+
+@pyxir.register_op_support_check("DPUCAHX8L-u280", "Pad")
+def pad_op_support(X: XLayer, bXs: List[XLayer], tXs: List[XLayer]) -> bool:
+    """ Check whether we can execute the provided Padding operator
+        on the DPUCAHX8L-u280 target """
+    return is_padding_supported(X, bXs, tXs)
+
+@pyxir.register_op_support_check("DPUCAHX8L-u280", "Pooling")
+def pooling_op_support(X: XLayer, bXs: List[XLayer], tXs: List[XLayer]) -> bool:
+    """ Check whether we can execute the provided Pooling operator
+        on the DPUCAHX8L-u280 target """
+    return is_pooling_supported(
+        X=X,
+        bXs=bXs,
+        tXs=tXs,
+        channel_parallel=32,
+        bank_depth=4096,
+        max_pool_min_kernel=2,
+        max_pool_max_kernel=8,
+        max_pool_min_stride=1,
+        max_pool_max_stride=8,
+        avg_pool_min_kernel=1,
+        avg_pool_max_kernel=8,
+        avg_pool_min_stride=1,
+        avg_pool_max_stride=8,
+        max_pool_kernel_valid=[ 2, 3, 5, 7, 8],
```

Comment:
max_pool_kernel_valid isn't necessary for this DPU: https://www.xilinx.com/html_docs/vitis_ai/1_3/compiling_model.html#ztl1570696058091